components/update-overview/update-overview.js

```diff
@@ -1,57 +1,104 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import { ButtonSecondary, UpdateExtractSmall, UpdatesExtractLarge } from '../'
+import { ButtonSecondary, Topics, UpdateExtractSmall, UpdatesExtractLarge } from '../'
 
 class UpdateOverview extends Component {
   constructor(props) {
     super(props)
 
     this.state = {
+      activeTopic: 'All',
+      topics: this.getTopics(props.updatesData),
       pageOffset: 1,
       pageSize: 6,
       loading: false,
+      filteredUpdates: this.filterUpdates(props.updatesData, 'All')
     }
+
+    this.changeTopicHandler = this.changeTopicHandler.bind(this)
+    this.getTopics = this.getTopics.bind(this)
+    this.filterUpdates = this.filterUpdates.bind(this)
+    this.handleClick = this.handleClick.bind(this)
+    this.incrementPageOffset = this.incrementPageOffset.bind(this)
+    this.isHighlightedUpdate = this.isHighlightedUpdate.bind(this)
+    this.hasSelectedTopic = this.hasSelectedTopic.bind(this)
   }
 
-  handleClick = () => {
-    this.setState({
-      loading: true,
-    })
-    setTimeout(this.incrementPageOffset, 800)
+  handleClick() {
+    this.setState({ loading: true })
+    setTimeout(this.incrementPageOffset, 400)
   }
 
-  incrementPageOffset = () => {
+  incrementPageOffset() {
     this.setState({
       pageOffset: this.state.pageOffset + 1,
       loading: false,
     })
   }
 
-  filterUpdates = () => {
-    const { data, updatesData } = this.props
-    let filteredUpdates = [...updatesData]
+  changeTopicHandler(value) {
+    const { updatesData } = this.props
+    const filteredUpdates = this.filterUpdates(updatesData, value)
 
-    data.highlights.forEach(highlight => {
-      // filter out items that are already shown as highlights
-      filteredUpdates = filteredUpdates.filter(update => update.title !== highlight.title)
-    })
+    this.setState({
+      activeTopic: value,
+      filteredUpdates: filteredUpdates,
+      pageOffset: 1,
+     })
+  }
+
+  getTopics(updates) {
+    const allTopics = updates.map(update => update.category.name)
+    const uniqueTopics = [...new Set(allTopics)]
+    uniqueTopics.unshift('All')
 
-    return filteredUpdates
+    return uniqueTopics
+  }
+
+  filterUpdates(updates, topic) {
+    return updates
+      .filter(update => this.isHighlightedUpdate(update))
+      .filter(update => this.hasSelectedTopic(update, topic))
+  }
+
+  isHighlightedUpdate(update) {
+    const { data } = this.props
+    return !data.highlights.some(highlight => highlight.id === update.id)
+  }
+
+  hasSelectedTopic(update, topic) {
+    if (topic === 'All')
```

Comment:
a nit pick but you could write this shorter like:
if(topic === 'All') return true
return update.category.name === topic